CHANGELOG.md

```diff
@@ -4,7 +4,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Next Version]
-- Implement reward-aware preference optimization.
 - Added TRT-LLM support in PPO. This can be enabled by doing `trainer.ppo.trt_llm.enable=True`. There is also aview(-1) reshard option to reshard out pipeline parallelism during inference for further speedup via `trainer.ppo.trt_llm.reshard=True`.
 - PPO algorithm will now detect if the sample sequence is ended, and if so zero out the gradient of the samples that did not stop properly.
 - Added critic warmup in PPO with the flag `trainer.ppo.critic_warmup_steps`.
```

Comment:
Added critic warmup to the PPO with the flag `trainer.ppo.critic_warmup_steps`.

Reply:
Thanks. 

For my own notes, converting your suggestion to a github suggestion so I remember what to change:
```suggestion
- Added critic warmup to the PPO with the flag `trainer.ppo.critic_warmup_steps`.
```

Reply:
Suggestions applied